### PR TITLE
TASK1: Implement scissors game

### PR DESCRIPTION
**Pull request by lavrinovu4**
Implemented scissors game according task-01.

### CODE REVIEW

**Comment by alexposukhov** [comment]
Could you please split this pull request into two commits?
First with initial empty files, second with game implementation.

File: 01_git/scissors/scissors.c [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+static char *options[] = {
+    "rock\0",
+    "paper\0",
+    "scissors\0"
```
Comment (ekovalyov) [inline 01_git/scissors/scissors.c:8]: What is the reason for additional \0 in string. Compiler add it by default.

File: 01_git/scissors/scissors.c [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+};
+
+static char *options_user[] = {
+    ['r'] = "rock\0",
+    ['p'] = "paper\0",
+    ['s'] = "scissors\0"
```
Comment (ekovalyov) [inline 01_git/scissors/scissors.c:14]: Unused \0

**Review by ekovalyov — APPROVED** [review]

File: 01_git/scissors/scissors.c [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+    // Use current time as seed
+    srand(time(0));   
+
+    printf("Please choose: rock (r) - paper (p) - scissors (s). q - exit.\n");
+
+    while(1) {
+
+        // wait for user choise
+        scanf("%s", buf);
```
Comment (ekovalyov) [inline 01_git/scissors/scissors.c:54]: scanf("%s") is not safe.
it is possible to fix it but setting maximum number of chars.

**Review by ekovalyov — APPROVED** [review]

File: 01_git/scissors/scissors.c (hunk 1 above)
Comment (lavrinovu4) [reply 01_git/scissors/scissors.c]: Indeed, it is not needed, Fixed in https://github.com/Kernel-GL-HRK/gl_kernel_procamp_2021/pull/15/commits/b96e5a0237c2a6c2f7a8cc99f7905f573e7b7697

File: 01_git/scissors/scissors.c (hunk 2 above)
Comment (lavrinovu4) [reply 01_git/scissors/scissors.c]: Fixed in https://github.com/Kernel-GL-HRK/gl_kernel_procamp_2021/pull/15/commits/b96e5a0237c2a6c2f7a8cc99f7905f573e7b7697

File: 01_git/scissors/scissors.c (hunk 3 above)
Comment (lavrinovu4) [reply 01_git/scissors/scissors.c]: Fixed in https://github.com/Kernel-GL-HRK/gl_kernel_procamp_2021/pull/15/commits/b96e5a0237c2a6c2f7a8cc99f7905f573e7b7697

**Pull request merged**